docs/getting-started/08-connect-external-application.md

```diff
@@ -0,0 +1,278 @@
+
+   The system returns a response similar to this one:
+
+   ```bash
+   https://connector-service.{CLUSTER_DOMAIN}/v1/applications/signingRequests/info?token=h31IwJiLNjnbqIwTPnzLuNmFYsCZeUtVbUvYL2hVNh6kOqFlW9zkHnzxYFCpCExBZ_voGzUo6IVS_ExlZd4muQ==
+   ```
+
+   Save this output with the token to the clipboard, as you will need it in the next steps.
+
+   > **CAUTION:** The token included in the output is valid for 5 minutes.
+
+  </details>
+  <details>
+  <summary label="console-ui">
+  Console UI
+  </summary>
+
+1. Return to the general view in the Console UI by selecting **Back to Namespaces**.
+2. Go to **Integration** > **Applications/Systems** and select **Create Application**.
+3. Set the application's name to `commerce-mock` and select **Create** to confirm the changes.
+
+  Wait for the application to have the `SERVING` status.
+
+4. Open the newly created application and select **Connect Application**.
+5. Copy the whole URL string with the token by clicking **Copy to Clipboard** and select **OK** to close the pop-up box.
+
+  > **CAUTION:** The token included in the URL is valid for 5 minutes.
+
+</details>
+</div>
+
+### Connect events
+
+To connect events from Commerce mock to the microservice, follow these steps:  
+
+1. Once in the `order-service` Namespace, go to **Configuration** > **API Rules** > **commerce-mock** in the left navigation panel.
+2. Open the link under the **Host** column to access Commerce mock.
+3. Click **Connect**.
+4. Paste the previously copied URL string with the token. Confirm by selecting **Connect** and wait until the application gets connected.
+5. Select **Register All** on the **Local APIs** tab or just register **SAP Commerce Cloud - Events** to be able to send events.
+
+Once registered, you will see all Commerce mock APIs and events available under the **Remote APIs** tab.
+
+> **TIP:** Local APIs are the ones available within the mock application. Remote APIs represent the ones registered in Kyma.
+
+### Expose events in the Namespace
+
+To expose events in the `orders-service` Namespace, first create an ApplicationMapping CR to bind the application to the Namespace. Then, provision the Events API in the Namespace using the ServiceInstance CR.
+
+Follow these steps:
+
+<div tabs name="expose-events-in-namespace" group="connect-external-application">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Create an [ApplicationMapping CR](/components/application-connector/#custom-resource-application-mapping) and apply it to the cluster:
+
+  ```yaml
+  cat <<EOF | kubectl apply -f -
+  apiVersion: applicationconnector.kyma-project.io/v1alpha1
+  kind: ApplicationMapping
+  metadata:
+    name: commerce-mock
+    namespace: orders-service
+  EOF
+  ```
+
+2. List available ServiceClass CR in the `orders-service` Namespace. Under the `EXTERNAL-NAME` column, find one with the `sap-commerce-cloud-events-*` prefix.
```

Comment:
```suggestion
2. List the available ServiceClass CRs in the `orders-service` Namespace.
```

I believe this part `Under the `EXTERNAL-NAME` column, find one with the `sap-commerce-cloud-events-*` prefix.` should move to after the first command, as you find it only after listing all the CRs and immediately copy the name, which is what the next line says.